index.html

```diff
@@ -0,0 +1,102 @@
+        </ul>
+        
+    </nav>
+    <div id='popup'></div>
+    <header>
+        <h5 class="first-text">"Hello! Dear Businessman"</h5>
+        <h1 class="second-text">We have the softare you need to take your business 
+            to next level</h1>
+        <p class="paragraph">We have develop the system you were looking for, wich will help you
+            track your loans, payments and reports with all the useful data you need.
+        </p>
+    </header>
+    <section class="services">
+        <h2 class="services-title">services</h2>
+        <hr>
+        <ul class="services-wrapper">
+            <li class="services-item">
+              <i class="service-icon fas fa-calculator"></i>
+              <p class="service-title">automatic calculate interest</p>              
+              <p class="service-text">
+                our system gives you the interest that you'll receive automatically.
+              </p>
+            </li>
+            <li class="services-item">
+                <i class="service-icon fas fa-dollar-sign"></i>
+              <p class="service-title">automatic calculate mora</p>              
+              <p class="service-text">
+                if you client doesnt pay at time the system calculates a due interest. 
+              </p>
+            </li>
+            <li class="services-item">
+                <i class="service-icon fas fa-file-alt"></i>
+              <p class="service-title">payments reports</p>              
+              <p class="service-text">
+                you'll be able to see the payments that you clients do in a daily, weekly and monthly basis.
+              </p>
+            </li>
+            <li class="services-item">
+                <i class="service-icon fas fa-print"></i>
+              <p class="service-title">print your reports</p>              
+              <p class="service-text">
+                you'll be able to print the payments reports whenever you need it.
+              </p>
+            </li>
+            <li class="services-item">
+                <i class="service-icon fas fa-sign-in-alt"></i>
+              <p class="service-title">access to your data</p>              
+              <p class="service-text">
+                you'll be able to access to your business data whenever you want and wherever you are.
+              </p>
+            </li>
+        </ul>
+        <button class="service-btn">more info</button>
+    </section>
+    <section class="testimonials">
+        <h2 class="testimonials-title">testimonials</h2>
+        <hr class="separator-testimonials">
+        <ul class="testimonials-wrapper" id='testimonial-section'>
+            
+            
+        </ul>
+        
+          <div class="btn-testimonials" onclick="showAllTestimonials()">
+            <p id="btn-text">more</p><span id="arrow">&#711;</span>
+          </div>
+        
+    </section>
+    <footer>
```

Comment:
Add the partner's icons on the top of the footer and that applies only to the home page
![right2](https://user-images.githubusercontent.com/58376020/124035278-9f263780-d9fc-11eb-833b-f76e5c17ee52.png)

Reply:
Issue resolved 👍